src/wiki/templatetags/wiki_tags.py

```diff
@@ -197,3 +198,9 @@ def wiki_settings(name):
 @register.filter
 def starts_with(value, arg):
     return value.startswith(arg)
+
+@register.simple_tag(takes_context=True)
+def url_replace(context, **kwargs):
```

Comment:
The name of this tag is a bit hard to understand - especially when the tag doesn't come with any docstring. Would suggest calling it `url_querystring_replace` ?